drivers/timer/sam0_rtc_timer.c

```diff
@@ -179,11 +199,11 @@ int z_clock_driver_init(struct device *device)
 {
 	ARG_UNUSED(device);
 
+	/* Set up bus clock and GCLK generator. */
```

Comment:
This comment is no longer correct, as it's in front of the MCLK block which has no GCLK.

Reply:
I will clarify the comment.